### PR TITLE
updates for compatibility with up next addon, more generic approach

### PR DESCRIPTION
**Pull request by im85288**
Hi,

I'm replacing the next up addon with a new 'up next' one and made a couple of changes so that the communication was more generic. A couple of little changes like you now sending across the play_info and the id to call. 

The up next addon still needs skinned properly (sualfred is working on it) but works well and simply has the one popup. For info on the data changes you can see here [addon Integration](https://github.com/im85288/service.upnext/wiki/Addon-Integration)

### CODE REVIEW

**Comment by im85288** [comment]
Oh and you can remove the setting of the ```embyid``` in the ```plotoutline``` as I now just assume that if there is data there to process it is for the next episode to play who's info is in  ```playinfo```

File: resources/lib/functions.py [hunk 1]
```diff
@@ -1085,7 +1085,7 @@ def PLAY(params):
     play_info["media_source_id"] = media_source_id
     play_info["use_default"] = use_default
     log.info("Sending embycon_play_action : {0}", play_info)
-    send_event_notification("embycon_play_action", play_info)
```
Comment (faush01) [inline resources/lib/functions.py]: Why this change? Are you trying to be more generic?
But this is the main play event for the addon.
If another addon sends this event EmbyCon will try to play the file.
I dont think you can make this generic and allow other addons to send this as it will break this addon.

**Comment by im85288** [comment]
As agreed a new PR has been submitted which makes this one redundant
